packages/embed/webpack.config.js

```diff
@@ -20,20 +24,26 @@ const baseConfig = {
           transform: (content, path) => {
             const result = sass.renderSync({
               file: path,
+              outputStyle: isProd ? 'compressed' : 'expanded',
             })
 
             return result.css.toString()
           },
         },
       ],
     }),
+    new webpack.DefinePlugin({
+      'process.env': {
+        CSS_URL: JSON.stringify(isProd ? '://embed.typeform.com/next/css/' : './lib/css/'),
+      },
+    }),
   ],
 }
 
 const npmConfig = {
   ...baseConfig,
   entry: './src/index.ts',
-  devtool: 'inline-source-map',
+  devtool: isProd ? 'inline-source-map' : false,
```

Comment:
Are you sure about this setting? It might be helpful for debugging purposes.

Reply:
You are right, it should be the other way. Keep source maps for development, remove it only in production. It is a question of 13kB vs 50kB in lib size :)